recipes/flake8-builtins/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "flake8-builtins" %}
+{% set version = "0.2" %}
+{% set sha256 = "d426403d6c73e19af6e9f720da8329809ccd454d046872e674c3b2bd0ac9666b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - flake8
+
+test:
+  commands:
+    - "flake8 --version | grep flake8_builtins:"
```

Comment:
It's checking that flake8 can find the plugin, which is what really matters--better IMO than an import test, since you're never really going to import this. I did this before in conda-forge/pep8-naming-feedstock.